### PR TITLE
Add #metadata-version-2 feature designator (#491).

### LINKED ISSUE #491: Add #metadata-version-2 feature designator
(as an extension of ``#metadata``) to cover new metadata features added to TTML2, e.g., ``ttm:item`` element and ``@ttm:alt``.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #491.

### CODE REVIEW

**Comment by nigelmegitt** [comment]
Built version can be found at https://rawgit.com/w3c/ttml2/issue-0491-metadata-version-2-build/index.html

**Comment by skynavga** [comment]
@nigelmegitt please check the newest commit

**Comment by nigelmegitt** [comment]
@skynavga the changes look fine to me.

File: spec/ttml2.xml [hunk 1]
```diff
@@ -28902,7 +28963,18 @@ where <emph>chunking</emph> refers to the subdivision of resource content into c
...
 </ulist>
 </div3>
-
+<div3>
+<head>Metadata Attributes</head>
+<p>The following attribute was added in order to expand metadata functionality:</p>
+<ulist>
+<item><p><att>ttm:alt</att></p></item>
+</ulist>
+<ednote>
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: This ednote can now be removed (and needs to be)

File: spec/ttml2.xml [hunk 2]
```diff
@@ -21381,6 +21381,55 @@ present metadata information is considered to be implementation
 dependent.</p>
 </note>
 </div3>
+<div3 id="feature-metadata-alt">
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: This feature should be removed now that we have decided not to include the ttm:alt attribute.

**Comment by skynavga** [comment]
Need to remove reference to ``@ttm:alt``, now agreed to not support.

**Comment by skynavga** [comment]
Removed all remaining references to ttm:alt and #metadata-alt. @nigelmegitt please check.

File: spec/ttml2.xml (hunk 2 above)
Comment (skynavga) [reply spec/ttml2.xml]: Fixed.

File: spec/ttml2.xml (hunk 1 above)
Comment (skynavga) [reply spec/ttml2.xml]: Fixed.

**Pull request merged**